offload/test/mapping/target_has_device_addr.c

```diff
@@ -66,8 +66,9 @@ void zoo() {
   short **xpp = &xp[0];
 
   x[1] = 111;
-#pragma omp target data map(tofrom : xpp[1][1]) use_device_addr(xpp[1][1])
-#pragma omp target has_device_addr(xpp[1][1])
+#pragma omp target data map(tofrom : xpp[1][1]) map(xpp[1])                    \
```

Comment:
`use_device_addr(xpp[1][1:1])` is non-conforming from OpenMP's perspective, as it violates this restriction from OpenMP 5.1 2.14.2:
```
If a list item in a use_device_addr clause is an array section, the base expression must be a
base language identiﬁer.
```
The same is present in OpenMP 6.0 as well.

It is supposed to apply to array-elements/pointer-dereferences etc. as well. That is to be clarified in a future version of OpenMP).